src/data/faq.json

```
@@ -2171,66 +2171,304 @@
+                },
+                {
+                    "term": "API",
+                    "anchor": "api",
+                    "description": "APIs (also called application programming interface) are program parts that are made available by a software system to other programs for connection to the system."
+                }
+            ],
             "B": [
+                {
+                    "term": "Backup",
+                    "anchor": "backup",
+                    "description": "Backups refer to versions of a project at a certain point in time in the past, which can later be used to restore a project to its previous state."
+                },
                 {
                     "term": "Business Rules",
                     "anchor": "business_rules",
-                    "description": "For the introduction of <a href='#glossary_digital_certificate'>Digital COVID Certificates</a>, the eHealth network of the European Commission and the EU member states agreed that every country compiles the individual rules which have to be checked for transnational travelling. These \"Business Rules\" have two categories:<ul><li>Invalidation Rules of the Country of Administration, which decide under which circumstances a digital certificate is rejected</li><li>Acceptance Rules of the Country of Arrival, which must be fulfilled to enter the country</li></ul>"
+
+                    "description": "For the introduction of <a href='#glossary_digital_certificate'>Digital COVID Certificates</a>, the eHealth network of the European Commission and the EU member states agreed that every country compiles the individual rules which have to be checked for transnational travelling. These \"Business Rules\" have two categories:<ul><li>Invalidation Rules of the Country of Administration, which decide under which circumstances a <a href='#glossary_digital_certificate'>certificate</a> is rejected</li><li>Acceptance Rules of the Country of Arrival, which must be fulfilled to enter the country</li></ul>"
                 }
             ],
             "C": [
                 {
                     "term": "Certificate",
                     "anchor": "certificate",
                     "description": "See <a href='#glossary_digital_certificate'>Digital Certificate</a>"
+                },
+                {
+                    "term": "Check-in",
+                    "anchor": "check_in",
+                    "description": "Check-in is a process by which a person registers at a certain location or event. In order to offer the CWA's contact tracking check-in function, it is necessary that a QR code is published in the entrance area or when entering an event, which CWA users scan in order to digitally check-in."

```

No "tracking" only "tracing", or?

```suggestion
                    "description": "Check-in is a process by which a person registers at a certain location or event. In order to offer the CWA's contact tracing check-in function, it is necessary that a QR code is published in the entrance area or when entering an event, which CWA users scan in order to digitally check-in."

```

---

@Ein-Tim 
Good catch!
In https://www.coronawarn.app/en/faq/#check_in_implementation it starts with:
"Presence Tracing - in CWA also referred to as Event Registration ..." so the Glossary should align with the FAQ article.

---

Will be changed in the next commit, thanks!